lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -23,6 +23,40 @@ function find(query, context) {
   return result;
 }
 
+/** @typedef {{lighthouseResult: LH.Result}} PSIResponse */
+
+const PSI_KEY = 'AIzaSyAjcDRNN9CX9dCazhqI4lGR7yyQbkd_oYE';
+
+/**
+ * @param {string} url
+ * @return {Promise<PSIResponse>}
+ */
+function callPSI(url) {
+  const psiUrl = new URL('https://www.googleapis.com/pagespeedonline/v5/runPagespeed');
+  /** @type {Record<string, string | string[]>} */
+  const params = {
+    key: PSI_KEY,
+    url,
+    category: [
+      'performance',
+      'accessibility',
+      'seo',
+      'best-practices',
+      'pwa',
+    ],
+    strategy: 'mobile',
+    utm_source: 'Lighthouse Chrome Extension',
+  };
+  Object.entries(params).forEach(([key, value]) => {
+    const values = Array.isArray(value) ? value : [value];
+    for (const singleValue of values) {
+      psiUrl.searchParams.append(key, singleValue);
+    }
+  });
+
+  return fetch(psiUrl.href).then(res => res.json());
```

Comment:
Building script doesn't allow for es6, hence no async/await here. It also injects some Promise polyfills (hey ... we don't like that anymore, right?), so it seems like a larger issue for another day.